nhais-adaptor/Pipfile

```diff
@@ -11,6 +11,7 @@ integration-adaptors-common = {editable = true,path = "./../common"}
 fhirclient = "*"
 PyInquirer = "*"
 
+
```

Comment:
Not a major issue, but this PR shouldn't really change the NHAIS adaptor's Pipfile at all (even if it's only whitespace).

Reply:
@Philip-Woods stop messing about with my Pipfile dude 😂

Reply:
God dammit how did I manage that 🤦‍♂